chains/solana/contracts/programs/ccip-offramp/src/lib.rs

```diff
@@ -65,6 +47,34 @@ pub mod ccip_offramp {
         Ok(())
     }
 
+    /// Initializes the CCIP Offramp Config account.
+    ///
+    /// The initialization of the Offramp is responsibility of Admin, nothing more than calling these
+    /// initialization methods should be done first.
+    ///
+    /// # Arguments
+    ///
+    /// * `ctx` - The context containing the accounts required for initialization of the config.
+    /// * `svm_chain_selector` - The chain selector for SVM.
+    /// * `enable_execution_after` - The minimum amount of time required between a message has been committed and can be manually executed.
+    pub fn initialize_config(
+        ctx: Context<InitializeConfig>,
+        svm_chain_selector: u64,
+        enable_execution_after: i64,
+    ) -> Result<()> {
+        let mut config = ctx.accounts.config.load_init()?;
+        require!(config.version == 0, CcipOfframpError::InvalidInputs); // assert uninitialized state - AccountLoader doesn't work with constraint
+        config.version = 1;
+        config.svm_chain_selector = svm_chain_selector;
+        config.enable_manual_execution_after = enable_execution_after;
+        config.owner = ctx.accounts.authority.key();
+        config.ocr3 = [
+            Ocr3Config::new(OcrPluginType::Commit as u8),
+            Ocr3Config::new(OcrPluginType::Execution as u8),
```

Comment:
It seems odd to me that we'd hit the stack limit since `config` is zero copy, perhaps it's these two lines? Does it make any difference if you make the new method `const fn` or if you inline the code here?

Try also directly assigning `config.ocr3[0]` and `config.ocr3[1]`. It must either be this array allocation, or the `Ocr3Config::new` calls

Reply:
Hey Blaz, I thought so too, but at first I tried even commenting out the entire code of the method, and it still failed once the feature flag was deactivated. It seems like just the size of the context and the instruction data is already big enough to make it fail. 
Also, based on https://github.com/anza-xyz/agave/issues/1186#issue-2278967670, it seems like the `init` constraint already eats away a decent amount of the stack, meaning that `Can't have more than 3 accounts to init within the same instruction atm.`